articles/time_to_say_goodby.md

```diff
@@ -0,0 +1,55 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie
+issue: 8
+title: Time To Say Goodby
+---
+# {{page.title}}
+
+Time to Say Goodbye - What Should be Considered for a Successful Project Completion? 
+
+Setting up a project in an efficient way is not an easy task: To share visions of the project or of the project manager, to ensure that all are aligned and have a common understanding of how the project will be managed. It is nowadays a good practice of most project managers to have a kick off workshops, not only 1h but one day with with teambuilding and vision sharpening, not only once but depending on the project duration in regular intervals. 
```

Comment:
Kürzungspotential: "To share visions of the project or of the project manager, .." -> "To share the vision of the project, ..."
Kürzungspotential: "It is nowadays a good practice of most project managers to have a kick off workshops" -> "It is nowadays a good practice to have a kick off workshops"